docs/index.md

```diff
@@ -0,0 +1,17 @@
+# Internal - Build and Release Process for a New Version of the Docs and Index for Devdocs
+This Build and Release process provides instructions to set up and configure the [EOS Documentation Portal](https://docs.eosnetwork.com).
+
+|Process | Description |
+|---|---|
+|[Install Docsgen Software](https://github.com/eosnetworkfoundation/docsgen/blob/fix-devrel%23181/docs/Docsgen-Install-Process.md) | Provides guidelines to install the software necessary for generationg documents. Provides information to install and setup documentation generation, such as software requirements, dependencies, links to instructions on how to install the software, and scripts and instructions for initial setup of web document content.|
```

Comment:
Use relative links here. To resolve links that were in `./README.md` but are now in `docs/index.md`, you need to consider that `index.md` is at `./docs`, whereas README.md is at the root (`./`). *So a link to say `docs/1.md` referenced from `README.md` should now just read `./1.md` if referenced from `docs/index.md` because the file will be at the same folder*. Also, your absolute URL links to your remote branch `fix-devrel-181`, which will be deleted once the PR gets merged to `main`, and at that point the link will get broken - in short: not a good idea. Please change to relative links.